firstvoices/backend/views/dictionary_views.py

```diff
@@ -52,3 +57,116 @@ def get_queryset(self):
             )
         else:
             return DictionaryEntry.objects.none()
+
+
+@extend_schema_view(
+    list=extend_schema(
+        description="Returns a word of the day for the given site.",
+        responses={
+            200: DictionaryEntryDetailSerializer,
+            403: OpenApiResponse(description="Todo: Not authorized for this Site"),
+            404: OpenApiResponse(description="Todo: Site not found"),
+        },
+    ),
+)
+class WordOfTheDayView(
+    FVPermissionViewSetMixin,
+    SiteContentViewSetMixin,
+    mixins.ListModelMixin,
+    viewsets.GenericViewSet,
+):
+    """
+    Picks and returns word-of-the-day for a given site.
+    """
+
+    http_method_names = ["get"]
+    serializer_class = DictionaryEntryDetailSerializer
+    pagination_class = None
+
+    @staticmethod
+    def get_suitable_queryset(site_slug):
+        # Case 1. Check if there is a word assigned word-of-the-day date of today
+        today = datetime.today()
+        selected_word = WordOfTheDay.objects.filter(
+            site__slug=site_slug, date=today
+        ).first()
+        if selected_word is not None:
```

Comment:
could this condition just be `if selected_word:` ? Same with case 3

Reply:
It could! I don't mind either way